### PR TITLE
MEPTS-714: M&Q Categoria 13 Numerator Part 4

### PR DESCRIPTION
**Pull request by amosh101**

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/QualityImprovement2020CohortQueries.java [hunk 1]
```diff
@@ -4098,4 +4098,362 @@ public CohortDefinition getMQC13P3NUM_I() {
...
+            + "     INNER JOIN encounter e ON e.patient_id = p.patient_id  "
+            + "     INNER JOIN obs o ON o.encounter_id = e.encounter_id  "
+            + "     WHERE  "
+            + "         p.voided = 0 AND e.voided = 0  "
+            + "             AND o.voided = 0  "
+            + "             AND e.encounter_type = ${6}  "
+            + "             AND e.location_id = :location  "
+            + "             AND e.encounter_datetime BETWEEN :startDate AND :endDate  "
+            + "             AND o.concept_id = ${856}  "
+            + "             AND o.value_numeric > 1000  "
```
Comment (psmatsinhe) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/QualityImprovement2020CohortQueries.java]: @amosh101 same problem as the other ticket

**Review by psmatsinhe — APPROVED** [review]

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/QualityImprovement2020CohortQueries.java [hunk 2]
```diff
@@ -4172,11 +4172,181 @@ public CohortDefinition getMQ13P4B() {
...
+            + "        INNER JOIN encounter e ON e.patient_id = p.patient_id  "
+            + "        INNER JOIN obs o ON o.encounter_id = e.encounter_id  "
+            + "        WHERE  "
+            + "            p.voided = 0 AND e.voided = 0  "
+            + "                AND o.voided = 0  "
+            + "                AND e.location_id = :location  "
+            + "                AND e.encounter_type = ${6}  "
+            + "                AND e.encounter_datetime BETWEEN :startDate AND :endDate  "
+            + "                AND o.concept_id = ${856}  "
+            + "                AND o.value_numeric > 1000  "
```
Comment (psmatsinhe) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/QualityImprovement2020CohortQueries.java:4338]: This is not on the description

**Review by psmatsinhe — CHANGES_REQUESTED** [review]

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/QualityImprovement2020CohortQueries.java (hunk 2 above)
Comment (amosh101) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/QualityImprovement2020CohortQueries.java:4338]: Updated into correct query as well

**Review by psmatsinhe — APPROVED** [review]
Looks good to me

**Pull request merged**